src/cli.rs

```diff
@@ -33,6 +18,10 @@ pub enum Opt {
         /// Path to optional config file
         config_file: Option<String>,
 
+        #[structopt(short, long)]
+        /// Mode to run the proxy in
```

Comment:
Yep, great point.  I got lazy because it was a change in direction to use a string; originally I had an enum but those don't work well with structopt and settings.